### DISCUSSION TITLE
Adding console output for failed tests on dashboard

### DISCUSSION THREAD

**Discussion by jarretmoses** [cypress-io/cypress#14565] [Cypress Cloud]
I was curious if there's any intention to capture and visually show the browser's console out put (atleast on failed tests) in order to help facilitate debugging. I feel like this could be relatively helpful.

**Comment by bahmutov** [comment]
Agree, it would be, we have open proposals to implement this:
- https://github.com/cypress-io/cypress/issues/700
- https://github.com/cypress-io/cypress/issues/448
For now, I suggest using this plugin https://github.com/flotwig/cypress-log-to-output
